src/pages/Login/LoginPage.tsx

```diff
@@ -1,50 +1,47 @@
 import { useDispatch, useSelector } from "react-redux";
-import { RootState } from "../common/store";
-import { requestLogin, resetErrorMessage } from "./Login.slice";
-import { IRequestLoginAction } from "./Login.slice";
+import { RootState } from "../../app/common/store";
+import { requestLogin, resetErrorMessage } from "./LoginPage.slice";
+import { IRequestLoginAction } from "./LoginPage.slice";
 import { Field, Form, Formik } from "formik";
 import { FormTextField } from "../../shared/FormTextField";
 import { LayoutContainer } from "../../shared/LayoutContainer";
 import { Routes } from "../../shared/Routes.constants";
 import { Redirect } from "react-router";
-import { LocalStorage } from "../common/LocalStorage";
-import makeStyles from "@mui/styles/makeStyles";
-import createStyles from "@mui/styles/createStyles";
+import { LocalStorage } from "../../app/common/LocalStorage";
 import {
   Alert,
   Button,
   Card,
   CardActions,
   CardContent,
   CircularProgress,
-  Theme,
+  Stack,
 } from "@mui/material";
-
+import { styled } from "@mui/material/styles";
 //Refernce: https://github.com/creativesuraj/react-material-ui-login/blob/master/src/components/Login.tsx
-const useStyles = makeStyles((theme: Theme) =>
-  createStyles({
-    container: {
-      display: "grow",
-      flexWrap: "wrap",
-      alignItems: "center",
-      justifyContent: "center",
-      width: "50%",
-      margin: `${theme.spacing(0)} auto`,
-      left: "50%",
-      top: "50%",
-    },
-    loginBtn: {
-      marginTop: theme.spacing(2),
-      flexGrow: 1,
-    },
-    card: {
-      marginTop: theme.spacing(10),
-    },
-  })
+
+const LoginForm = styled(Form)(
+  ({ theme }) => `
+  display: inline-block;
+  flex-wrap: wrap;
+  align-items: center;
+  justify-content: center;
+  width: 50%;
+  margin: ${theme.spacing(2)} auto;
+  left: 50%;
+  top: 50%;
+`
 );
 
+const FancyButton = styled(Button)`
+  flex-grow: 1;
+`;
```

Comment:
This button is still not named based on the function, but it can be modified in the next PR.

Reply:
I swore I changed this. I just pushed a change to fix it.